packages/headless/src/app/store.ts

```diff
@@ -33,6 +35,7 @@ export function configureStore<Reducers extends ReducersMapObject>({
         (state as {history?: unknown}).history
           ? {...state, history: '<<OMIT>>'}
           : state,
+      name: name || 'coveo-headless',
```

Comment:
I'd prefer to set the default at a higher level. In the near future, there will be several engine types, which I think could have different defaults. `coveo-headless-search`, `coveo-headless-recommendation`, `coveo-headless-case-assist` etc.

Making `name` required will signal that a good default should be passed rather than relying on the default inside `store.ts`.

Reply:
That works. I'll make name a required field for the store and make the good default passed down from a higher level.